src/freenet/clients/http/WelcomeToadlet.java

```diff
@@ -583,7 +588,20 @@ public void handleMethodGET(URI uri, HTTPRequest request, ToadletContext ctx) th
         this.writeHTMLReply(ctx, 200, "OK", pageNode.generate());
     }
 
-	private void putFetchKeyBox(ToadletContext ctx, HTMLNode contentNode) {
+    private void putCoreOperationsInterface(ToadletContext ctx, HTMLNode contentNode) throws IOException {
+		  HTMLNode coreOperationsBox = ctx.getPageMaker().getInfobox("infobox-normal", l10n("nodeOperationLabel"), contentNode, "node-operation", true);
+      coreOperationsBox.addAttribute("id", "coreNodeOperation");
```

Comment:
This line seems to have escaped its intended indentation! 🙂

Reply:
Ah, yes! I now matched it to its indented intention ☺ - thank you!